Allow to configure directory where the main .revision file is stored …

martin-sucha opened the pull request:
…on server

banago commented:
@martin-sucha Thanks for the contribution. Can you please update readme.md file accordingly about this change. Also, why do you think this change is needed?

martin-sucha commented:
@banago I've updated the readme, please let me know in case anything needs to be clarified.

I added this option because my hosting provider only allows me to write to two SFTP subdirectories (one is inside document root, the other is not). I have those two subdirectories inside my git repo, so that both can be updated at the same time. This also means I have to store the revision file in one of the subdirectories (I don't have write access to the SFTP root).

One more thing - I'm not sure whether this option works with absolute paths (this depends on how the FTP library works). I only tried with relative name of the subdirectory, which works for me.

banago commented:
@martin-sucha makes sense. Thanks for the contribution.
